deploy/advanced/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: aws-secret-operator
+  labels:
+    app: aws-secret-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: aws-secret-operator
+  template:
+    metadata:
+      labels:
+        name: aws-secret-operator
+        app: aws-secret-operator
+    spec:
+      serviceAccountName: aws-secret-operator
+      containers:
+        - name: aws-secret-operator
+          # Replace this with the built image name
+          image: mumoshu/aws-secret-operator:v0.1.0
+          ports:
+          - containerPort: 60000
+            name: metrics
+          command:
+          - aws-secret-operator
+          imagePullPolicy: Always
+          readinessProbe:
+            exec:
+              command:
+                - stat
+                - /tmp/operator-sdk-ready
+            initialDelaySeconds: 4
+            periodSeconds: 10
+            failureThreshold: 1
+          env:
+            - name: WATCH_NAMESPACE
+              value: ""
```

Comment:
Setting to blank allows the CRD to check all namespaces (assuming RBAC is in place)